android/quest/src/test/java/org/smartregister/fhircore/quest/ui/patient/details/QuestPatientDetailScreenTest.kt

```diff
@@ -153,7 +153,7 @@ class QuestPatientDetailScreenTest : RobolectricTest() {
       .onNodeWithTag(PATIENT_NAME)
       .assertExists()
       .assertIsDisplayed()
-      .assertTextEquals("John Doe, M, 21y")
+      .assertTextEquals("John Doe, M, 22y")
```

Comment:
why is this changed?

Reply:
Its being failing in PR,

Reply:
do you know why it's failing? is there a birthday that's set to December 27th or something like that?

Reply:
I'm not suggesting we should revert this change, I'm trying to understand if this is a brittle test that will fail every time the year changes.

If that's true, we'd want to be able to load the date being used and do some math to convert that to the number of years used here